src/transformers/configuration_utils.py

```diff
@@ -337,7 +337,9 @@ def get_config_dict(cls, pretrained_model_name_or_path: str, **kwargs) -> Tuple[
         elif os.path.isfile(pretrained_model_name_or_path) or is_remote_url(pretrained_model_name_or_path):
             config_file = pretrained_model_name_or_path
         else:
-            config_file = hf_bucket_url(pretrained_model_name_or_path, filename=CONFIG_NAME, use_cdn=False)
+            config_file = hf_bucket_url(
+                pretrained_model_name_or_path, filename=CONFIG_NAME, use_cdn=False, mirror=None
+            )
```

Comment:
Don't you want to be able to pass a `mirror` option in the from_pretrained kwargs?

Reply:
Nah. We can preserve some stats with the config IMO.